packages/extensions/openapi-to-typespec/src/interfaces.ts

```diff
@@ -63,13 +63,31 @@ export interface TypespecResource {
   response: TypespecDataType;
 }
 
+export interface AadOauth2AuthFlow {
+  kind: "AadOauth2Auth";
+  scopes: string[];
```

Comment:
I think some of these interfaces might need the auth url and token url

Reply:
I couldn't see those in M4, it seems to assume the login.microsoft url